modules/sc-mesh-secure-deployment/src/2_0/features/jamming/options.py

```diff
@@ -1,85 +1,24 @@
-import os
-import re
 from typing import List
-from log_config import logger
 
 VALID_CHANNELS = [36, 40, 44, 48, 149, 153, 157, 161]
 
-
 class Options:
     def __init__(self):
-        self.jamming_osf_orchestrator: str = 'fd01::1'
+        self.jamming_osf_orchestrator: str = 'fd04::1'
```

Comment:
OSF addressing has changed so this default is probably not correct anymore

Reply:
Thanks for pointing this out, this argument should be configured for each specific device, so the current set value was for the particular device I was working on, I'll look into the new addressing